deploy/kubedirector/kubedirector.hpe.com_kubedirectorapps_crd.yaml

```diff
@@ -96,8 +96,6 @@ spec:
                         type: string
                       isDashboard:
                         type: boolean
-                      hasAuthToken:
-                        type: boolean
```

Comment:
Are these some accidental deletions from this CRD? Merge issues?

Reply:
Yes.